docs/API_ACCESS.md

```diff
@@ -0,0 +1,62 @@
+The Search API is implemented with GraphQL. This document explains how to access the service to send requests.
+
+## Pre-requisite
+Red Hat Advanced Cluster Management installed on your OpenShift cluster.
+
+## Locate the Search API endpoint
+### From within the cluster
+If your application is deployed in the cluster, and it has the correct permissions, it can send requests to https://search-search-api.open-cluster-management.svc:4010/searchapi/graphql
+
+### From outside the cluster
+By default, the Search API is not exposed. Follow [this topic](https://docs.openshift.com/container-platform/4.9/networking/routes/secured-routes.html) to create a secure route to the service `search-search-api` in the `open-cluster-management` namespace.
+
+```
+# NOTE: this sample requires an insecure connection from the client.
+oc create route passthrough search-api --service=search-search-api -n open-cluster-management
+```
+
+The route should look like this:  `https://search-api-open-cluster-management.apps.[your-ocp-hostname]/searchapi/graphql`
+
+## Authenticating the requests
+
+The Search API needs the user (or ServiceAccount) token to ensure that results only contain resources for the authorized user.
+
+- A user can obtain their token with `oc whoami -t`
+- For a service account the token is in a secret. [This article](https://www.ibm.com/docs/en/cloud-paks/cp-management/2.0.0?topic=kubectl-using-service-account-tokens-connect-api-server) explains how to get the token.
```

Comment:
Should we instead say that:

1. Create a service account. 
2. Give it role/rolebinding so it can READ from the service endpoint
3. Then use the token from this service NEWLY created service account.

As is mentioned here: https://docs.openshift.com/container-platform/4.9/authentication/understanding-and-creating-service-accounts.html

Reply:
Updated, thanks.